content/partials/general/events/_batched_events.textile

```diff
@@ -38,6 +38,24 @@ minimize. View batched @message@ payload example
   }
   ```
 
+h5(#batch-example-message-decoding). Decoding batched messages
+
+Messages sent "over the realtime service":/realtime/channels are automatically decoded into "@Messages@":/realtime/types#message object by the Ably client library. With webhooks you need to to do this explicitly, using "@Message.fromEncodedArray@":/realtime/messages#message-from-encoded-array on the @data.messages@ array, or "@Message.fromEncoded@":/realtime/messages#message-from-encoded or an individual member of that array. This will transform them into an array of "@Messages@":/realtime/types#message (or in the case of @fromEncoded@, an individual "@Message@":/realtime/types#message). This has several advantages, e.g.:
```

Comment:
`"@Messages@":/realtime/types#message object` => `"@Message@":/realtime/types#message objects` I think?

Reply:
`or an individual member of that array` => `on an individual member of that array